src/jsonapi-vuex.js

```diff
@@ -571,49 +573,64 @@ const checkAndFollowRelationships = (state, getters, records, seenState) => {
 }
 
 // Follow relationships and expand them into _jv/rels
-const followRelationships = (state, getters, record, seenState = {}) => {
-  // Copy item before modifying
-  const data = cloneDeep(record)
+const followRelationships = (state, getters, record, seen = []) => {
+  // Make a shallow copy of the object's keys (by reference - preserve getters).
+  // We can't add rels to the original object, otherwise Vue's watchers
+  // spot the potential recursion and throw an error
```

Comment:
Spot the potential loop, rather than recursion. Recursion is allowed, loops (or infinite recursion) aren't.